### PR TITLE
DOC: Tidied up EDS documentation

### PR DESCRIPTION
**Pull request by tjof2**
As mentioned by @dnjohnstone in #964, I've tidied up the structure and grammar of the EDS documentation page.

I don't think that all of the section headings are necessarily appropriate - would appreciate any input here.

### CODE REVIEW

File: doc/user_guide/eds.rst [hunk 1]
```diff
@@ -543,50 +561,81 @@ The obtained composition is in atomic percent, by default. However, it can be tr
...
     >>> weight_percent = hs.material.atomic_to_weight(atomic_percent)
 
-The zeta-factor method needs both the 'beam_current' (in nA) and the acquisition or dwell time (referred to as 'real_time' in seconds) in order to provide an accurate quantification. Both of the these parameters can be assigned to the metadata of the spectrum using:
+The reverse method is :py:func:`~.misc.material.weight_to_atomic`.
+
+The zeta-factor method needs both the 'beam_current' (in nA) and the acquisition
+or dwell time (referred to as 'real_time' in seconds) in order to obtain an accurate
+quantification. Both of the these parameters can be assigned to the metadata using:
 
 ..code-block:: python
```
Comment (francisco-dlp) [inline doc/user_guide/eds.rst]: Now that you're at it you could also fix the missing space here, it should be: `.. code-block:: python`

File: doc/user_guide/eds.rst [hunk 2]
```diff
@@ -543,50 +561,81 @@ The obtained composition is in atomic percent, by default. However, it can be tr
...
-The cross section method needs the 'beam_current', dwell time ('real_time') and probe area in order to provide accurate quantification. The 'beam_current' and 'real_time' can be set using :py:meth:`~._signals.eds_tem.EDSTEMSpectrum.set_microscope_parameters` as shown above. The 'probe_area' (in nm^2) can be defined in two different ways. Either, the probe diameter is narrower than the pixel width, which case the probe is being under-sampled and therefore an estimation of the probe area needs to be used. This can be added to the metadata by:
+The cross section method needs the 'beam_current', dwell time ('real_time') and probe area
+in order to obtain an accurate quantification. The 'beam_current' and 'real_time' can be set as shown above.
+The 'probe_area' (in nm^2) can be defined in two different ways.
+
+If the probe diameter is narrower than the pixel width, then the probe is being
+under-sampled and an estimation of the probe area needs to be used. This can be added
+to the metadata with:
 
 ..code-block: python
```
Comment (francisco-dlp) [inline doc/user_guide/eds.rst]: Now that you're at it you could also fix the missing space here, it should be: `.. code-block:: python`

File: doc/user_guide/eds.rst [hunk 3]
```diff
@@ -504,20 +502,39 @@ The background can be subtracted from the X-ray intensities with the :py:meth:`~
...
-   EDS spectrum with background subtraction markers.
+   EDS spectrum with background subtraction markers
 
 .. _eds_quantification-label:
 
-Quantification
+EDS Quantification
 --------------
 
 -.. versionadded:: 0.8
```
Comment (francisco-dlp) [inline doc/user_guide/eds.rst]: And also remove the `-` here.

File: doc/user_guide/eds.rst [hunk 4]
```diff
@@ -442,10 +436,8 @@ Data files used in the following examples can be downloaded using
     The sample and the data used in this section are described in
     D. Roussow et al., Nano Lett, 10.1021/acs.nanolett.5b00449 (2015).
```
Comment (francisco-dlp) [inline doc/user_guide/eds.rst]: And maybe add this paper to the bibliography as those cited below?

**Comment by francisco-dlp** [comment]
Other than the minor suggestions above (that actually involve issues that were already present, so fixing them, although it would be nice, it is not required to merge this), I think that this is ready to merge (unless a native English speak thinks otherwise).

Only for info: for any future contribution it would be better if you could work on a separate new branch in your repository i.e. not on the master branch.

**Comment by tjof2** [comment]
@francisco-dlp changes made - I realised the branch mistake after I'd already pushed it, apologies.

**Pull request merged**